### ISSUE TITLE
Training from scratch for sports 1-M

### ISSUE THREAD

**Issue by aliericcantona** [hx173149/C3D-tensorflow#67]
Hi,

So far,

* downloaded 30000 videos from sports 1-M dataset per category 60 videos. Then I decoded the videos grabbing up to 50 frames from each video. 
* saved them exactly like UCF101 dataset.
* Used the code to split to train and test
* changed the num_classes to 487
* as of now using the same crop_mean.npy file. 

When running the training code, prints out 0.00 accuracy even after 5000 steps. **I was wondering If I have missed anything.** 

BTW, I used the same code and trained based on UCF 101. It was successful.

**Comment by aliericcantona** [comment]
Actually I was able to train from scratch by changing number of images per iteration to 100.
The problem now is the accuracy on the training data is great but not on the test data. 
Any clue? May be the test/train split does not do a good job.

**Comment by Ai-is-light** [comment]
@aliericcantona could you show how do you to change number of images per iteration to 100? where did you change
